pages/kommun/[municipality].tsx

```diff
@@ -166,4 +181,21 @@ const Municipality = () => {
   )
 }
 
-export default Municipality
+type Query = {
+  query: {
+    municipality: string
+  }
+}
+
+export async function getServerSideProps({query}: Query) {
+  const { municipality } = query
+  const res = await fetch(
+    `http://klimatkollen.vercel.app/api/municipality/${municipality}`,
```

Comment:
Kan vi lägga `http://klimatkollen.vercel.app/` som en ENV-variable istället? https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env

Vi kommer behöva den på mer än ett ställe :)